edxval/serializers.py

```diff
@@ -205,3 +211,40 @@ def update(self, instance, validated_data):
             course_video.save()
 
         return instance
+
+class VideosPagination(pagination.BasePagination):
+    """
+    Paginator for APIs in edx-val.
+    """
+    invalid_page_message = 'Invalid page.'
+
+    def paginate_queryset(self, queryset, page_no=1, page_size=20):
+        """
+        Paginate the queryset and returns the interable data
+        for the given page and page_size
+        """
+        paginator = Paginator(queryset, page_size)
+
+        try:
+            self.page = paginator.page(page_no)
+        except InvalidPage as exc:
+            msg = self.invalid_page_message.format(
+                page_number=page_no, message=six.text_type(exc)
+            )
```

Comment:
What are you trying to format here ? `self.invalid_page_message` does't includes `page_number` & `message`. Are you expecting `self.invalid_page_message` from parent class. ?